### DISCUSSION TITLE
Node v14 LTS

### DISCUSSION THREAD

**Discussion by josteph** [uNetworking/uWebSockets.js#537] [Q&A] [answered]
Hi there!

Just recently tried with node v14.16.0, the node version doesn't seem to be supported?
The `process.versions.modules` holds value of `87`, but only `88` is available.

**Comment by ghost** [comment, marked as answer]
Node.js 14 is definitely supported and working. It's not 87, it is 83. You're most likely talking about Electron.

**Reply by josteph** [reply]
Ah that's right.n Any chance to use it together with Electron?

**Reply by ghost** [reply]
There's extensive text written on the subject
